recipes/asyncqt/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asyncqt" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8b1507c968c85cf0b7eee5d2a887162d38af15fb8c5b1ec25beed6025d7383ac
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - PyQt5
```

Comment:
This library works with PyQt5 or PySide2, decided to use PyQt5 here just for testing

Reply:
You need to change it to :
```
pyqt 5
```

Reply:
Or just let it unpinned and it will get the last version of ``pyqt``

Reply:
Hmmm it failed because importing `QtGui` also tried to import OpenGL... checking if I can use `PySide2` then.